raccoongang/edx-platform
# Fix problem with auto-saving changes in user profile interface

**Description:** UI fix in user profile interface
**Youtrack:** https://youtrack.raccoongang.com/issue/GREENHOST-70
**Testing instructions:**

1. Open LMS interface - got to /account/settings URL
2. Make changes to SELECT field
3. Correct behavior: "You changes has been saved" message appears
4. Incorrect: there is not message. Message appears only after clicking on page.

**Reviewers:**
- [x] @AndreyLykhoman 
- [ ] @sidorovdmitry 
- [x] @dgamanenko 

**Merge checklist:**

**Post merge:**

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.
